src/Components/ContentDelivery/ContentDelivery.js

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react'
+import './ContentDelivery.scss'
+import ReactPlayer from 'react-player'
```

Comment:
So sweet that there is this ReactPlayer! Nice job finding this @rwilliams659

Reply:
Yes, that makes this wildly convenient.